Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -117,12 +116,11 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
 
-    [self setupNavigationController];
```

Comment:
Q: why was this one relocated? (If there's some initialization that needs to happen before, mind adding a note in a comment?)

Reply:
Looking at it, I think that I moved the wrong thing actually.  setupNavigationController was coming after  configureNavigationControllerToolbar which created the buttons for the toolbar.  I moved that to viewDidLoad with the other configuration methods and put setupNavigationController back where it was so that it all fit consistently with the rest of the code flow